schema_forms/mammo_form.py

```diff
@@ -1,191 +1,142 @@
-        self.mammo_mass_density.data = mammography.mammo_mass_density
-        self.mammo_calcification_present.data = mammography.mammo_calcification_present
-        self.mammo_calc_number.data = mammography.mammo_calc_number
-        self.mammo_calc_location_right_breast.data = mammography.mammo_calc_location_right_breast
-        self.mammo_calc_location_left_breast.data = mammography.mammo_calc_location_left_breast
-        self.mammo_calc_depth.data = mammography.mammo_calc_depth
-        self.mammo_calc_dist.data = mammography.mammo_calc_dist
-        self.mammo_calcification_type.data = mammography.mammo_calcification_type
-        self.mammo_calcification_diagnosis.data = mammography.mammo_calcification_diagnosis
-        self.mammo_arch_present.data = mammography.mammo_arch_present
-        self.mammo_arch_location_right_breast.data = mammography.mammo_arch_location_right_breast
-        self.mammo_arch_location_left_breast.data = mammography.mammo_arch_location_left_breast
-        self.mammo_arch_depth.data = mammography.mammo_arch_depth
-        self.mammo_arch_dist.data = mammography.mammo_arch_dist
-        self.mammo_assym_present.data = mammography.mammo_assym_present
-        self.mammo_assym_location_right_breast.data = mammography.mammo_assym_location_right_breast
-        self.mammo_assym_location_left_breast.data = mammography.mammo_assym_location_left_breast
-        self.mammo_assym_depth.data = mammography.mammo_assym_depth
-        self.mammo_assym_type_right_breast.data = mammography.mammo_assym_type_right_breast
-        self.mammo_assym_type_left_breast.data = mammography.mammo_assym_type_left_breast
-        self.mammo_intra_mammary_lymph_nodes_present.data = mammography.mammo_intra_mammary_lymph_nodes_present
-        self.mammo_intra_mammary_lymph_nodes_description.data = mammography.mammo_intra_mammary_lymph_nodes_description
-        self.mammo_lesion.data = mammography.mammo_lesion
-        self.mammo_lesion_right_breast.data = mammography.mammo_lesion_right_breast
-        self.mammo_lesion_left_breast.data = mammography.mammo_lesion_left_breast
-        self.mammo_asso_feature_skin_retraction.data = mammography.mammo_asso_feature_skin_retraction
-        self.mammo_asso_feature_nipple_retraction.data = mammography.mammo_asso_feature_nipple_retraction
-        self.mammo_asso_feature_skin_thickening.data = mammography.mammo_asso_feature_skin_thickening
-        self.mammo_asso_feature_trabecular_thickening.data = mammography.mammo_asso_feature_trabecular_thickening
-        self.mammo_asso_feature_axillary_adenopathy.data = mammography.mammo_asso_feature_axillary_adenopathy
-        self.mammo_asso_feature_architectural_distortion.data = mammography.mammo_asso_feature_architectural_distortion
-        self.mammo_asso_feature_calicifications.data = mammography.mammo_asso_feature_calicifications
-        self.mammo_birad.data = mammography.mammo_birad
+        return bson
 
+    def from_bson(self, p):
```

Comment:
does from_bson have to be defined for each model so from_bson_mammo; from_bson_biopsy etc?